code/modules/events/brain_trauma.dm

```diff
@@ -17,6 +17,8 @@
 			continue
 		if(H.stat == DEAD) // What are you doing in this list
 			continue
+		if(HAS_TRAIT(H, TRAIT_SPECIAL_SPAWNED))
+			continue
```

Comment:
This should already be covered by the job flag check, no? Unless you're assigning your spawns jobs?

Reply:
i do not actually know